docs/examples/simple_examples.rst

```diff
@@ -0,0 +1,174 @@
+******************************
+Simple scintillometry examples
+******************************
+
+Consider observing a pulsar using a single radio dish.
+Let's first assume that the pular is stationary.
+In this basic scenario, the pulsar's radiation reaches the Earth via a direct
+line of sight, and the dynamic spectrum of the observation will look rather
+plain. In the secondary spectrum, there will be a single non-zero point at the
+origin.
+
+.. plot::
+
+    from astropy import units as u
+    import matplotlib.pyplot as plt
+    from example_figure import gen_magnifs, example_figs
+
+    theta = [0.] << u.mas
+    magnifs = gen_magnifs(theta)
+
+    example_figs(theta, magnifs, mu_eff=0.*u.mas/u.yr, screens=False,
+                 velocity=False)
+
+A scattering screen
+===================
+
+Now introduce a scattering screen somewhere in the space between the pulsar and
+Earth. This screen, a one-dimensional lens, will give rise to point-like
+scattering images of the pulsar. As a first step, we consider the effect of a
+single scattering image. In addition to the direct line of sight, there will
+now be a scattered beam via which radiation from the pulsar reaches Earth.
+
+The radiation going via the
```

Comment:
Super-nitpick, but if could you split sentences into lines of roughly equal lines?